src/components/Profile/RenderFavorites.js

```diff
@@ -0,0 +1,89 @@
+import { StyleSheet, Text, TouchableOpacity, View } from "react-native";
+import React from "react";
+import { Avatar, ListItem } from "react-native-elements";
+import HandleFavoriteButton from "./HandleFavoriteButton";
+
+export const renderType = {
+  GENRE: 0,
+  ANIME: 1,
+};
```

Comment:
If I'm not wrong the button type and render type is the same so it might be better if the const is centralized